genie/genie-cadvisor-client.go

```diff
@@ -179,3 +182,14 @@ func GetCNSOrderByNetworkBandwith(cAdvisorURL string) (string, error) {
 	fmt.Fprintf(os.Stderr, "CAdvisor Client response = %v\n", res)
 	return res, nil
 }
+
+/**
+Returns cAdvisor Address to collect network usage parameters
+	- conf : Netconf info having genie configurations
+*/
+func getCAdvisorAddr(conf NetConf) string {
+	if conf.CAdvisorAddr == "" {
```

Comment:
We should check and trim redundant spaces, if any, entered by the user.

Reply:
Fixed